--- bin/gpi_launch	2015-09-18 10:59:02.000000000 -0700
+++ bin/gpi_launch	2015-09-18 10:58:49.000000000 -0700
@@ -26,8 +26,8 @@

 import sys, os

-# GPI_PREFIX = '/opt/anaconda1anaconda2anaconda3' # ANACONDA
-GPI_PREFIX, _ = os.path.split(os.path.dirname(os.path.realpath(__file__)))
+GPI_PREFIX = '/opt/anaconda1anaconda2anaconda3' # ANACONDA
+# GPI_PREFIX, _ = os.path.split(os.path.dirname(os.path.realpath(__file__)))
 GPI_LIB_DIR = os.path.join(GPI_PREFIX, 'lib')
 if GPI_LIB_DIR not in sys.path:
     sys.path.insert(0, GPI_LIB_DIR)
